handle_http.go

```diff
@@ -478,6 +478,10 @@ func DownloadHTTP(transfer TransferDetails, dest string, token string) (int64, e
 	// Check the error real quick
 	if resp.IsComplete() {
 		if err := resp.Err(); err != nil {
+			if errors.Is(err, grab.ErrBadLength) {
+				grab.ErrBadLength = errors.New("Local copy of file is larger than remote copy")
```

Comment:
Why this assignment?  Doesn't seem necessary to override the text in the other module itself... can you keep the type but provide only `err` with the new message?

Reply:
Fixed with new commit!